### PR TITLE
docs: add section for RAC, with installation steps

### PR DESCRIPTION
**Pull request by Aceix**
image

### CODE REVIEW

File: pro/rac.rst [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+   sudo apt install remote-client
+
+
+******************
+Quick Start
+******************
+
+To use RAC, you will need to have a Netmaker server running and have a user account on that server. You will also need to have a client gateway set up on the server that you have access to.
+
+NOTE: RAC currently works with non-admin users
```
Comment (abhishek9686) [inline pro/rac.rst:38]: this is not a limitation, the idea is to only work with non-admin users

File: pro/rac.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+For Linux (Debian/Ubuntu), you can also use the following command to download the latest version:
+
+.. code-block:: 
+
+   curl -sL 'https://apt.netmaker.org/remote-client/gpg.key' | sudo tee /etc/apt/trusted.gpg.d/remote-client.asc
+   curl -sL 'https://apt.netmaker.org/remote-client/debian.deb.txt' | sudo tee /etc/apt/sources.list.d/remote-client.list
+   sudo apt update
+   sudo apt search remote-client  # to see available versions
+   sudo apt install remote-client
+
```
Comment (abhishek9686) [inline pro/rac.rst:30]: Probably mention that remote-client will be available in the app section, or basically mention how to launch the app

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: pro/rac.rst [hunk 3]
```diff
@@ -28,6 +28,9 @@ For Linux (Debian/Ubuntu), you can also use the following command to download th
    sudo apt search remote-client  # to see available versions
    sudo apt install remote-client
 
+For Mac, You can download the mac installer from the fileserver link above and run it.
+
+For Windows, you can download the remote-access-client_86.msi installer and run it to install on your windows machine.
```
Comment (abhishek9686) [inline pro/rac.rst:33]: @theguy951357 on mac and windows, shouldn't they be run as root form terminal?
can you mention how to run the app?

File: pro/rac.rst (hunk 1 above)
Comment (Aceix) [reply pro/rac.rst]: addressed

File: pro/rac.rst (hunk 2 above)
Comment (Aceix) [reply pro/rac.rst:30]: addressed

File: pro/rac.rst [hunk 4]
```diff
@@ -38,9 +40,8 @@ Quick Start
 
 To use RAC, you will need to have a Netmaker server running and have a user account on that server. You will also need to have a client gateway set up on the server that you have access to.
 
-NOTE: RAC currently works with non-admin users
-
 Check :ref:`this section <pro/pro-users:adding users>` on how to create a non-admin user.
+RAC can work with both admin and non-admin users but is more suited for non-admins who want to gain access to private resources on the network.
```
Comment (abhishek9686) [inline pro/rac.rst:44]: RAC is for non-admin users who want to gain remote access to the network, this also provides admins fine-grained control over users in the network by attaching/removing them from a client gateway

File: pro/rac.rst [hunk 5]
```diff
@@ -30,7 +30,7 @@ For Linux (Debian/Ubuntu), you can also use the following command to download th
 
 Following the above instructions, you can run RAC from your Linux desktop environment launcher or from the command line using the `remote-client` command.
 
-For Mac, You can download the mac installer from the fileserver link above and run it. Because the app needs to be run as root, when you open the application from the applications folder, it will run a script that opens the application with elevated privileges.
+For Mac, You can download the mac installer from the fileserver link above and run it. Because the app needs to be run as root, open a teminal and cd into the Applications directory. Enter the command ``sudo su``. You can also use the command ``sudo -i``. Then enter ``./NetmakerRemoteClient`` to run the program.
```
Comment (abhishek9686) [inline pro/rac.rst:34]: command to run the app on mac `sudo /Applications/NetmakerRemoteClient.app/Contents/MacOS/remote-client`

Comment (theguy951357) [reply pro/rac.rst:34]: done

**Review by abhishek9686 — APPROVED** [review]

**Pull request merged**